bank_integration/scripts/payment_entry_list.js

```
@@ -0,0 +1,93 @@
+        "online_payment_status",
+        "party_bank_ac_no",
+        "party_bank",
+        "pay_now",
+        "paid_from",
+    ],
+    onload(listview) {
+        frappe.realtime.off("eval_js");
+        frappe.realtime.off("payment_success_bulk");
+
+        frappe.realtime.on("eval_js", function (message) {
+            eval(message);
```

Biome (2.4.7)

[error] eval() exposes to security risks and performance issues.

(lint/security/noGlobalEval)